src/cases/components/case-hearings-list/case-hearings-list.component.ts

```diff
@@ -77,6 +77,9 @@ export class CaseHearingsListComponent implements OnInit {
     if (hearing.exuiDisplayStatus === EXUIDisplayStatusEnum.CANCELLATION_REQUESTED) {
       this.hearingStore.dispatch(new fromHearingStore.LoadHearingRequest(hearing.hearingID));
       this.router.navigate(['/', 'hearings', 'request', 'hearing-cancellation-summary']);
+    } else if (hearing.exuiDisplayStatus === EXUIDisplayStatusEnum.CANCELLED) {
+      this.hearingStore.dispatch(new fromHearingStore.LoadHearingRequest(hearing.hearingID));
+      this.router.navigate(['/', 'hearings', 'request', 'hearing-cancelled-summary']);
```

Comment:
If this for view details, could we move this to hearings/view/hearing-cancelled-summary, as it doesn't request page flow and different journey, same as the above one hearings/view/hearing-cancellation-summary